src/pages/CreateProfile/Activities.tsx

```diff
@@ -0,0 +1,142 @@
+import React from "react";
```

Comment:
Should: ensure when the activity is selected that it is highlighted. Currently when I select an activity, it's only after I select the next activity that the previous one is highlighted

Reply:
This is something I noticed and it’s actually just an issue with having dev tools open, cos the state of hovered on remains. Try it not on mobile version